junit5-maven-consumer/pom.xml

```diff
@@ -43,6 +44,21 @@
 						<artifactId>junit-platform-surefire-provider</artifactId>
 						<version>${junit.platform.version}</version>
 					</dependency>
+					<dependency>
+						<groupId>org.junit.jupiter</groupId>
+						<artifactId>junit-jupiter-engine</artifactId>
+						<version>${junit.jupiter.version}</version>
+					</dependency>
+					<dependency>
+						<groupId>org.junit.vintage</groupId>
+						<artifactId>junit-vintage-engine</artifactId>
+						<version>${junit.vintage.version}</version>
+					</dependency>
+					<dependency>
+						<groupId>org.junit.platform</groupId>
+						<artifactId>junit-platform-runner</artifactId>
+						<version>${junit.platform.version}</version>
```

Comment:
You're right ... that dependency isn't needed.  It seemed to be an artifact of some of the experiments I ran trying to use the JUnitPlatform runner without putting so many classes on the test-compile classpath.  Those experiments ended up producing this issue in the JUnit 5 project: https://github.com/junit-team/junit5/issues/498.

I'll alter this PR to match the users guide tomorrow when I get into work.